tests/import_test.go

```diff
@@ -611,6 +616,15 @@ var _ = Describe("Importer Test Suite-Block_device", func() {
 			return strings.Contains(log, "attempting to create blank disk for block mode")
 		}, controllerSkipPVCCompleteTimeout, assertionPollInterval).Should(BeTrue())
 		Expect(err).ToNot(HaveOccurred())
+
+		err = utils.WaitForPersistentVolumeClaimPhase(f.K8sClient, pvc.Namespace, v1.ClaimBound, pvc.Name)
+		Expect(err).ToNot(HaveOccurred())
+	},
+		Entry("[test_id:4972] with first consumer", true),
+		Entry("with bind immediate annotation", false),
+	)
+	It("Should create blank raw image for block PV", func() {
```

Comment:
Code leftover, looks good otherwise. Thanks for the PR!

Reply:
whoops thx